docker/models/containers.py

```diff
@@ -140,6 +140,43 @@ def exec_run(self, cmd, stdout=True, stderr=True, stdin=False, tty=False,
             resp['Id'], detach=detach, tty=tty, stream=stream, socket=socket
         )
 
+    def exec_run2(self, cmd, stdout=True, stderr=True, stdin=False, tty=False,
```

Comment:
`run2` is certainly not a descriptive name. what about a switch like `stream` that controls the return value?